src/api/entityResolution/types.ts

```diff
@@ -16,15 +16,37 @@ export type DeleteEntityResolutionMappingParams = IDataSourceParams & {sourceNod
 export interface EntityResolutionMapping<
   T extends EntityResolutionRecordType = EntityResolutionRecordType
 > {
+  /**
+   * The node category of the mapped graph node.
+   */
   sourceNodeCategory: string;
+  /**
+   * Which kind of entity resolution record we map to.
+   */
   targetRecordType: T;
+  /**
+   * How do we map graph properties to entity resolution attributes.
+   */
   properties: EntityResolutionPropertyMapping<T>[];
 }
 
 export interface EntityResolutionPropertyMapping<
   T extends EntityResolutionRecordType = EntityResolutionRecordType
 > {
+  /**
+   * The node category of the graph node on which the property is picked. If it is undefined, it
+   * means the property belongs to the main mapped node, identified by the source node category of
+   * the parent mapping. Otherwise, the property belongs to an adjacent node, identified by this
+   * category.
+   */
+  sourceNodeCategory?: string;
```

Comment:
A more explicit field name could be `sourceAdjacentNodeCategory`. It's pretty verbose, but it has the merit of being explicit.

A note about the wording: `adjacent` seems more accurate than `neighboring`, given its our current API, see https://doc.linkurious.com/server-sdk/latest/apidoc/#api-Node-getAdjacentNodes

Reply:
+1 for sourceAdjacentNodeCategory

Reply:
OK, changed to `sourceAdjacentNodeCategory`